### DISCUSSION TITLE
When version 1.2.6 expires, there is no retry button available

### DISCUSSION THREAD

**Discussion by hackgsl** [rustdesk/rustdesk#8748] [General]
After updating version 1.2.6 today, it was found that there was only one button in the confirmation box that popped up when the controlled end exceeded the time limit. Clicking it directly closed the remote control.
Previously, version 1.2.3 had a retry button, which could be clicked directly to connect without the need to reopen the main interface.

**Comment by rustdesk** [comment]
screenshot please? what accurate message is it?

**Reply by hackgsl** [reply]
![PixPin_2024-07-18_14-37-14](https://github.com/user-attachments/assets/36be4b28-1c53-4470-a31a-d862045640dd)
![PixPin_2024-07-18_14-52-33](https://github.com/user-attachments/assets/7d73ae5c-a9e1-4cb8-b5a0-75aa01557b62)
Additionally, a new issue has been discovered where the time cannot be modified for 'automatically closing inactive sessions'.

**Reply by hackgsl** [reply]
![PixPin_2024-07-18_15-16-21](https://github.com/user-attachments/assets/dbb4ce7a-8994-4132-a568-f161cb8d3385)

Version 1.2.3 has a "reconnect" button, which can be clicked to link again.
V1.2.3版本是有“重连”按钮，点击即可再次链接。

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/8750
https://github.com/rustdesk/rustdesk/pull/8749

**Comment by hackgsl** [comment]
This problem occurs again in update 1.3.2, rollback to 1.3.1 normal, uninstall and reinstall 1.3.2 failed to resolve.

**Reply by rustdesk** [reply]
@21pages

**Reply by rustdesk** [reply]
Intended or not? please confirm with AnyDesk

**Reply by 21pages** [reply]
Intended

**Reply by rustdesk** [reply]
Show AnyDesk snapshot please

**Reply by 21pages** [reply]
AnyDesk always has "Retry", different mechanisms.

https://github.com/rustdesk/rustdesk/blob/040253b3199ee9fe95a4d91f16634a9504b83ce0/src/server/connection.rs#L716

![55210cdf3ab04ddea574217adcdbecb](https://github.com/user-attachments/assets/2b154a8e-11ae-44c0-994f-7c502ef3abcb)
![19e1f53b696360eb6753722d28ea145](https://github.com/user-attachments/assets/b642e0ec-c746-45ba-a838-783537a215be)

**Reply by hackgsl** [reply]
> Intended or not? please confirm with AnyDesk有意还是无意？请与 AnyDesk 确认

If no operation is performed for a long time and the specified timeout period is reached, a message is displayed indicating that the connection is disconnected due to no operation for a long time.
In this case, you cannot click Reconnect to quickly recover. You need to close the prompt box and open the RustDesk homepage again.
![PixPin_2024-11-05_09-04-11](https://github.com/user-attachments/assets/bf024055-fcc4-467d-8d45-46d05c2658af)

**Reply by hackgsl** [reply]
> Show AnyDesk snapshot please

The question submitted yesterday was tagged "wontfix". Does that mean that this is normal and that no changes are planned for future versions?
